.github/workflows/release-build-publish.yaml

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Build/Publish
```

Comment:
Can we use the same workflow name and filename convention as in these scripts?

```
name: Release Che Dashboard
```
-- https://github.com/eclipse/che-dashboard/blob/master/.github/workflows/release.yml#L1-L2

```
name: Release Che Workspace Loader
```
-- https://github.com/eclipse/che-workspace-loader/blob/master/.github/workflows/release.yaml#L1-L2

I notice now that the files are either .yml or .yaml, so I'll get the .yaml one renamed.